packages/nameguard-react/src/components/DisplayedName/DisplayedName.tsx

```diff
@@ -1,98 +1,37 @@
 import { Normalization } from "@namehash/nameguard";
-import React, { useEffect, useState, useRef } from "react";
-
-import { Tooltip } from "../..";
 import { ENSName } from "@namehash/ens-utils";
+import React from "react";
+import { Text } from "../Text/Text";
 
 interface DisplayedNameProps {
-  ensName: ENSName;
-  maxEnsNameDisplayWidth?: number;
+  name: ENSName;
+  stylingClasses?: string;
+  maxDisplayWidth?: number;
+  maxTooltipWidth?: number;
   displayUnnormalizedNames?: boolean;
   displayTooltipWhenNameClamps?: boolean;
 }
 
-const DEFAULT_MAX_ENSNAME_DISPLAY_WIDTH = 200;
```

Comment:
I feel like we should keep a custom default for this inside `DisplayedName` that is distinct from any default inside `Text`.